src/Mutation/MediaItemUpdate.php

```diff
@@ -45,7 +45,7 @@ public static function get_output_fields() {
 	}
 
 	public static function mutate_and_get_payload() {
-		function ( $input, AppContext $context, ResolveInfo $info ) {
+		return function ( $input, AppContext $context, ResolveInfo $info ) {
```

Comment:
@hsimah looks like this just wasn't being returned. Made the change and confirmed all tests pass locally. 

I'll review the code, but on initial scan I'm digging the change

Reply:
d'oh!